webpack/assets/javascripts/react_app/components/CPUCoresInput/cpuCoresInput.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../common/colors';
+
+.help-block-warning-icon {
```

Comment:
You can have 1 class instead of 2 that doing the same.
```scss
.help-block-icon {
  margin-right: 3px;
}
```

```jsx
<WarningTriangleIcon className="help-block-icon" />
<ErrorCircleOIcon className="help-block-icon" />
```